### PR TITLE
Feat/esp32s3 ble provisioning

### PR DESCRIPTION
**Pull request by hugoferrari**
Description
-----------

Test Steps
-----------

Checklist:
----------

- [ ] I have tested my changes. No regression in existing tests.
- [ ] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
```
Comment (Dazza0) [inline .vscode/c_cpp_properties.json:1]: @hugoferrari These are VS Code project configuration files. Is there any reason to commit these?

File: components/espressif__esp_secure_cert_mgr/.component_hash [hunk 2]
```diff
@@ -0,0 +1 @@
+0a7b7f1c26fbd6fd8f49cc90f39203e3ab9cf334bbbf288bede9db745c35892c
```
Comment (Dazza0) [inline components/espressif__esp_secure_cert_mgr/.component_hash:1]: Any reason to commit the [ESP Secure Certificate Manager](https://components.espressif.com/components/espressif/esp_secure_cert_mgr) files? The IDF component manager is designed to download the files to each project specifically.

File: main/Kconfig.projbuild [hunk 3]
```diff
@@ -1,4 +1,4 @@
-menu "Featured FreeRTOS IoT Integration"
+menu "Kelsus Camp 2024"
```
Comment (Dazza0) [inline main/Kconfig.projbuild:1]: @hugoferrari Any reason for this naming change?

**Review by Dazza0 — CHANGES_REQUESTED** [review]
@hugoferrari I'm not really understanding the purpose of these changes. Could you add a summary/description to this PR outlining what the goal of these changes were.

**Comment by aggarg** [comment]
I am closing this as this seems like an accidental PR. Please let us know if that is not the case.

**Pull request closed without merging**